Public/Src/Cache/ContentStore/Interfaces/FileSystem/AbsolutePath.cs

```diff
@@ -36,6 +36,26 @@ public class AbsolutePath : PathBase, IEquatable<AbsolutePath>
         /// </summary>
         public static int MinimumPathLength = OperatingSystemHelper.IsWindowsOS ? 2 : 1;
 
+        /// <summary>
+        /// Create an AbsolutePath for the root of a drive.
+        /// </summary>
+        public static AbsolutePath RootPath(char driveLetter = default(char))
+        {
+            if (OperatingSystemHelper.IsWindowsOS)
+            {
+                if (driveLetter == default(char))
+                {
+                    throw new ArgumentException(nameof(driveLetter));
+                }
+
+                return new AbsolutePath($"{driveLetter}:{System.IO.Path.DirectorySeparatorChar}{System.IO.Path.DirectorySeparatorChar}");
+            }
+            else
+            {
+                return new AbsolutePath(System.IO.Path.DirectorySeparatorChar.ToString());

```

Comment:
nit: I would extract this into a static property like DirectorySeparator for simplicities sake.